src/main/kotlin/no/nav/familie/tilbake/behandling/domain/Fagsak.kt

```diff
@@ -3,7 +3,7 @@ package no.nav.familie.tilbake.behandling.domain
 import no.nav.familie.tilbake.common.repository.Sporbar
 import org.springframework.data.annotation.Id
 import org.springframework.data.relational.core.mapping.Embedded
-import java.util.UUID
+import java.util.*
```

Comment:
Små ting, prøv å unngå wildcard import om det er mulig.

Reply:
Default code style for kotlin i intellij sier at java.util skal være * import. Hvis du synes det plagsomt, så hadde det vært fint om du oppdaterte familie-settings.